src/main/java/de/dafuqs/spectrum/mixin/FoxEntityMixin.java

```diff
@@ -11,7 +11,7 @@
 @Mixin(FoxEntity.class)
 public class FoxEntityMixin {
 
-    @ModifyReturnValue(method = "isSleeping", at = @At("RETURN"))
+    @ModifyReturnValue(method = "isSleeping()Z", at = @At("RETURN"))
```

Comment:
nit: but why though? It works as it currently is just fine, no issues here. Unless there is? Is it related to mod compat?

Reply:
It wasn't working correctly, so I tried adding this and I guess I never undid the change.